x-pack/test/apm_api_integration/trial/tests/service_maps/service_maps.ts

```diff
@@ -86,157 +86,18 @@ export default function serviceMapsApiTests({ getService }: FtrProviderContext)
-                      "span.destination.service.resource": "redis",
-                      "span.subtype": "redis",
-                      "span.type": "cache",
-                    },
-                  },
-                },
-                Object {
-                  "data": Object {
-                    "id": "opbeans-node~opbeans-java",
-                    "source": "opbeans-node",
-                    "sourceData": Object {
-                      "agent.name": "nodejs",
-                      "id": "opbeans-node",
-                      "service.environment": "ENVIRONMENT_NOT_DEFINED",
-                      "service.name": "opbeans-node",
-                    },
-                    "target": "opbeans-java",
-                    "targetData": Object {
-                      "agent.name": "java",
-                      "id": "opbeans-java",
-                      "service.environment": "ENVIRONMENT_NOT_DEFINED",
-                      "service.name": "opbeans-java",
-                    },
-                  },
-                },
-                Object {
-                  "data": Object {
-                    "agent.name": "rum-js",
-                    "id": "client",
-                    "service.environment": "ENVIRONMENT_NOT_DEFINED",
-                    "service.name": "client",
-                  },
-                },
-                Object {
-                  "data": Object {
-                    "agent.name": "nodejs",
-                    "id": "opbeans-node",
-                    "service.environment": "ENVIRONMENT_NOT_DEFINED",
-                    "service.name": "opbeans-node",
-                  },
-                },
-                Object {
-                  "data": Object {
-                    "id": ">redis",
-                    "label": "redis",
-                    "span.destination.service.resource": "redis",
-                    "span.subtype": "redis",
-                    "span.type": "cache",
-                  },
-                },
-                Object {
-                  "data": Object {
-                    "agent.name": "java",
-                    "id": "opbeans-java",
-                    "service.environment": "ENVIRONMENT_NOT_DEFINED",
-                    "service.name": "opbeans-java",
-                  },
-                },
-                Object {
-                  "data": Object {
-                    "id": ">postgresql",
-                    "label": "postgresql",
-                    "span.destination.service.resource": "postgresql",
-                    "span.subtype": "postgresql",
-                    "span.type": "db",
-                  },
-                },
-              ],
+          expect(status).to.be(200);
+          const environments = new Set();
+          body.elements.forEach((element: { data: Record<string, any> }) => {
+            environments.add(element.data['service.environment']);
+          });
+          expectSnapshot(environments).toMatchInline(`
```

Comment:
IMO, no need for this to be a snapshot test. It can be an assertion, that the only value in the set is `ENVIRONMENT_NOT_DEFINED`.

Also, can you add `expectSnapshot(body).toMatch()`? This will store a snapshot in a separate file.